frontend/amundsen_application/static/js/config/config-utils.ts

```diff
@@ -77,7 +80,45 @@ export function getResourceNotices(
   const { notices } = AppConfig.resourceConfig[resourceType];
 
   if (notices && notices[resourceName]) {
-    return notices[resourceName];
+    const thisNotice = notices[resourceName];
+    if (typeof thisNotice.messageHtml === 'function') {
+      thisNotice.messageHtml = thisNotice.messageHtml(resourceName);
+    }
+    return thisNotice;
```

Comment:
Sorry for the late comment, but I think we could do something like:
```
const withComputedMessage = (notice, resourceName) => {
if (typeof notice.messageHtml === 'function') {
      notice.messageHtml = notice.messageHtml(resourceName);
    }
    return notice;
}
```
And use it on line 115 as well.